### PR TITLE
changes a few words

### PR DESCRIPTION
**Pull request by ShiftyRail**
Thank you Microsoft 

All of the content we removed will be rehosted on the secret repo shortly :tm:

### CODE REVIEW

File: code/game/dna/genes/goon_disabilities.dm [hunk 1]
```diff
@@ -124,7 +124,6 @@
 	speech.message = replacetext(speech.message,"drink","water")
 	speech.message = replacetext(speech.message,"feminist","empowered woman")
 	speech.message = replacetext(speech.message,"i hate you","you're mean")
-	speech.message = replacetext(speech.message,"nigger","african american")
```
Comment (jknpj) [inline code/game/dna/genes/goon_disabilities.dm:127]: :thinking:

File: code/game/machinery/vending.dm [hunk 2]
```diff
@@ -2695,10 +2695,10 @@ var/global/num_vending_terminals = 1
 	pack = /obj/structure/vendomatpack/shoedispenser
 
 //HEIL ADMINBUS
-/obj/machinery/vending/nazivend
-	name = "\improper Nazivend"
-	desc = "A vending machine containing Nazi German supplies. A label reads: \"Remember the gorrilions lost.\""
-	icon_state = "nazi"
+/obj/machinery/vending/wehrmachtvend
+	name = "\improper wehrmachtvend"
+	desc = "A vending machine containing WW2 German supplies. A label reads: \"Remember the gorrilions lost.\""
```
Comment (boy2mantwicethefam) [inline code/game/machinery/vending.dm:2700]: might want to change the gorrilions just to be sure

File: code/datums/religions.dm [hunk 3]
```diff
@@ -662,7 +662,7 @@
 	bible_type = /obj/item/weapon/storage/bible/booze
 	male_adept = "LGBT Advocate"
 	female_adept = "LGBT Advocate"
-	keys = list("homosexuality", "faggotry", "gayness", "gay", "penis", "faggot", "cock", "cocks", "dick", "dicks")
+	keys = list("homosexuality", "faggotry", "gayness", "gay", "penis", "bundle of sticks", "cock", "cocks", "dick", "dicks")
```
Comment (ghost) [inline code/datums/religions.dm:665]: You missed the other bad word here

File: code/game/objects/structures/barsigns/vg.dm [hunk 4]
```diff
@@ -48,9 +48,9 @@
 	icon = "foreign"
 	desc = "The only bar you will ever need, comrade!"
 
-/datum/barsign/vg/nazi
+/datum/barsign/vg/wehrmacht
 	name = "Der Fuhrer's Trinken Haus"
```
Comment (Kurfursten) [inline code/game/objects/structures/barsigns/vg.dm:52]: Kaiser's Trinken Haus?

File: code/game/turfs/simulated/walls.dm [hunk 5]
```diff
@@ -270,7 +270,7 @@
 		var/obj/item/weapon/pickaxe/PK = W
 		if(!(PK.diggables & DIG_WALLS))
 			return
-		if(mineral == "diamond") //Nigger it's a one meter thick wall made out of diamonds
+		if(mineral == "diamond") //philistine it's a one meter thick wall made out of diamonds
```
Comment (Kurfursten) [inline code/game/turfs/simulated/walls.dm:273]: Let's all take a moment to appreciate how not OOP this is

File: code/modules/admin/verbs/deadsay.dm [hunk 6]
```diff
@@ -24,7 +24,7 @@
 	if (!msg)
 		return
 
-	var/rendered = "<span class='game deadsay'><span class='prefix'>DEAD:</span> <span class='name'>ADMIN([src.holder.fakekey ? pick("BADMIN", "hornigranny", "TLF", "scaredforshadows", "KSI", "Silnazi", "HerpEs", "BJ69", "SpoofedEdd", "Uhangay", "Wario90900", "Regarity", "MissPhareon", "LastFish", "unMportant", "Deurpyn", "Fatbeaver") : src.key])</span> says, <span class='message'>\"[msg]\"</span></span>"
```
Comment (Kurfursten) [inline code/modules/admin/verbs/deadsay.dm:27]: surely we can just axe this one entirely, or put in our own joke admin names... former admin list perhaps?

File: code/game/dna/genes/goon_disabilities.dm (hunk 1 above)
Comment (PJB3005) [reply code/game/dna/genes/goon_disabilities.dm:127]: Not taking any chances I'd say.

File: code/modules/admin/verbs/deadsay.dm (hunk 6 above)
Comment (ghost) [reply code/modules/admin/verbs/deadsay.dm:27]: That's the stealthmin nonsense so you can just replace it with "stealthed" or something... or put the name of the notemaster as the one true stealthmaster

File: code/modules/mob/living/simple_animal/head.dm [hunk 7]
```diff
@@ -29,8 +29,7 @@
...
-	"Ezekiel 25:17 ,The path of the righteous man is beset on all sides by the iniquities of the selfish and the tyranny of evil men. Blessed is he who in the name of charity and good will shepherds the weak through the valley of darkness, for he is truly his brother's keeper and the finder of lost children. And I will strike down upon thee with great vengeance and furious anger those who attempt to poison and destroy my brothers. And you will know my name is the Lord... when I lay my vengeance upon thee.",
-	"Did you notice a sign out in front of my house that said \"Dead Nigger Storage\"?")
+	"Ezekiel 25:17 ,The path of the righteous man is beset on all sides by the iniquities of the selfish and the tyranny of evil men. Blessed is he who in the name of charity and good will shepherds the weak through the valley of darkness, for he is truly his brother's keeper and the finder of lost children. And I will strike down upon thee with great vengeance and furious anger those who attempt to poison and destroy my brothers. And you will know my name is the Lord... when I lay my vengeance upon thee.")
```
Comment (Kurfursten) [inline code/modules/mob/living/simple_animal/head.dm:32]: not your fault, but comma out of place at the start of this

File: code/modules/clothing/head/misc.dm [hunk 8]
```diff
@@ -369,9 +369,9 @@
 	item_state = "panzercap"
 	flags = FPRINT
 
-/obj/item/clothing/head/naziofficer
+/obj/item/clothing/head/wehrmachtofficer
 	name = "Officer Cap"
-	desc = "Ein Hut von Offizieren in der Nazi-Partei getragen."
+	desc = "Ein Hut von Offizieren in der wehrmacht-Partei getragen."
```
Comment (Kurfursten) [inline code/modules/clothing/head/misc.dm:374]: just delete -Partei entirely

File: code/modules/mob/living/simple_animal/hostile/human/nazi.dm [hunk 9]
```diff
@@ -1,10 +1,10 @@
-/mob/living/simple_animal/hostile/humanoid/nazi
-	name = "\improper Nazi"
+/mob/living/simple_animal/hostile/humanoid/wehrmacht
+	name = "\improper wehrmacht"
 	desc = "Sieg Heil!"
```
Comment (Kurfursten) [inline code/modules/mob/living/simple_animal/hostile/human/nazi.dm:3]: Heil, Kaiser, dir!

File: code/modules/admin/verbs/deadsay.dm (hunk 6 above)
Comment (d3athrow) [reply code/modules/admin/verbs/deadsay.dm:27]: darkhrose...

Comment (Kammerjunk) [reply code/modules/admin/verbs/deadsay.dm:27]: that is a list of very old admins

Comment (Kurfursten) [reply code/modules/admin/verbs/deadsay.dm:27]: Assumed they were old goon or baymins, wew.

AFAIK not even one of these guys was still around when I started in late 2014

Comment (boy2mantwicethefam) [reply code/modules/admin/verbs/deadsay.dm:27]: At least one is a former /tg/ admin and that's scaredofshadows

**Comment by ShiftyRail** [comment]
there is another path

**Comment by tuypo1** [comment]
You can't just say that you are gonna have the removed content anyway, this isn't about advertisers or anything where they are happy as long as you hide it like the panda, this is about social control and if you let them know you are not actually acquiescing they aren't gonna care

**Comment by Exxion** [comment]
Dude, it's a ToS issue
Their ToS obviously says nothing about what you do on other sites

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 10]
```diff
@@ -830,13 +830,13 @@
 	reagents.add_reagent(TOXIN, 2)
 	bitesize = 2
 
-/obj/item/weapon/reagent_containers/food/snacks/faggot
-	name = "faggot"
+/obj/item/weapon/reagent_containers/food/snacks/meatball
```
Comment (DrSnips) [inline code/modules/reagents/reagent_containers/food/snacks.dm:833]: The greatest loss is definitely what-must-now-be-called-meatball. God speed round ball of mystery meat.

**Comment by tuypo1** [comment]
> Their ToS obviously says nothing about what you do on other sites

It's cute that you think that matters

**Comment by Exxion** [comment]
It's ridiculous you think it doesn't

**Comment by ghost** [comment]
> You can't just say that you are gonna have the removed content anyway, this isn't about advertisers or anything where they are happy as long as you hide it like the panda, this is about social control and if you let them know you are not actually acquiescing they aren't gonna care

we know for a fact that the only reason tg/hippie were taken down is because they were targeted by a coordinated report attack, github didn't suddenly start cracking down on things that go against random values for the sake of censorship

**Comment by tuypo1** [comment]
Sounds more like slow bureaucracy than anything else but I suppose it means as long as we don't piss off any discord groups or anything we will probably be fine

**Comment by Exxion** [comment]
I don't know where you got the idea that corporations care about anything other than money

**Comment by tuypo1** [comment]
Pattern recognition

**Comment by Exxion** [comment]
They don't, ever (at least for large corporations). If it looks like they do, it's because they determined that doing so would make them more money. But it's still always about appearances.
People who work there might care about other things, but the company as a whole doesn't.

**Comment by diegoxt45** [comment]
what a shame.

**Comment by ShiftyRail** [comment]
nothingburger + conflicts

**Pull request closed without merging**